ci-operator/config/codeready-toolchain/registration-service/OWNERS

```diff
@@ -0,0 +1,22 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexeykazakov
+- dipak-pawar
+- matousjobanek
+- michaelkleinhenz
```

Comment:
@nurali-techie Can you please remove `michaelkleinhenz`, as he is not part of openshift organisation. I have already asked him to do the necessary steps for it.

Reply:
`michaelkleinhenz` is now part of openshift org.